sigs/recommenders/CHARTER.md

```diff
@@ -33,24 +33,26 @@ Everybody with an interest in using TensorFlow to build recommendation systems a
 
 - SIG Recommenders [mailing list](https://groups.google.com/u/1/a/tensorflow.org/g/recommenders)
 - New TensorFlow github maintained by SIG Recommenders
-(github.com/tensorflow/recommenders-addons, to be created)
+(github.com/tensorflow/recommenders-addons)
```

Comment:
Suggestion for replacing the 2 lines above with:
´´´
 - GitHub repository maintained by SIG Recommenders: https://github.com/tensorflow/recommenders-addons
´´´